icu4c/source/tools/genrb/reslist.cpp

```diff
@@ -352,6 +353,17 @@ IntVectorResource::~IntVectorResource() {
 }
 
 void IntVectorResource::add(int32_t value, UErrorCode &errorCode) {
+    if (fCount == fSize) {
+        uint32_t* tmp = new uint32_t[2 * fSize];
```

Comment:
We can't `new[]` in library code, but this is genrb, and we are already `new[]`ing in this file, so it's fine.